packages/stacked_themes/lib/src/theme_manager.dart

```diff
@@ -60,6 +60,15 @@ class ThemeManager {
   /// Returns true if the ThemeMode is dark. This does not apply when you're using system as ThemeMode
   bool get isDarkMode => _selectedThemeMode == ThemeMode.dark;
 
+  /// Get currently selected theme
+  int? get selectedTheme {

```

Comment:
Could you change the name of this property to be more descriptive please. `selectedThemeIndex` would be better.